docs(refactor): tutorials MTA-1809 (#443)
scaleway/docs-content: tutorials/deploy-saas-application/index.mdx

  h1: Deploying a Software as-a-Service application with Scaleway
  paragraph: In this tutorial you will learn how to deploy a Software as-a-Service (SaaS) application, based on a SaaS application developed using Django Template.
tags: Saas software-as-a-Service software Django-Template
categories: 
  - compute
  - container-registry
dates:
  validation: 2022-02-28
  posted: 2021-03-09
---

In this tutorial you will learn how to deploy a Software as-a-Service (SaaS) application, based on a SaaS application developed using Django Template. You use the following Scaleway products:

* [**Object Storage**](https://www.scaleway.com/en/object-storage/), to store your static files
* [**Database as-a-Service**](https://www.scaleway.com/en/database/), for your data storage
* [**Container Registry**](https://www.scaleway.com/en/container-registry/), to store your Docker images
* [**Kubernetes Kapsule**](https://www.scaleway.com/en/kubernetes-kapsule/), for your production environment

> Along the way, you learn how to store environment variables with Kubernetes secrets and use ImagePullSecrets to pull your private images. We also give you some tips for going further with your deployed SaaS application, including how to expose it with a Scaleway Load Balancer.

Reviewer: ```suggestion
You will learn how to store environment variables with Kubernetes secrets and use ImagePullSecrets to pull your private images. We also give you some tips for going further with your deployed SaaS application, including how to expose it with a Scaleway Load Balancer.
```
